.coveragerc

```diff
@@ -1,15 +0,0 @@
-# .coveragerc to control coverage.py
```

Comment:
That's such a pain... it keeps getting deleted and I'm not sure what is causing it. I dont think I ran any `make` commands and it's still gone.

Reply:
I can help take a look next time this happens. Do you want to have some time looking at some git features to help spot this?

Reply:
TBH, I just forgot to look at the code changes since I had to submit the code a bunch of times.

`make test` doesn't run locally for me with `z-quantum-core` so the only way to see if some problems occurred is to submit a bunch of times and look at the results.

Reply:
I guess what I'm trying to say is, some help with figuring out why `make test` isn't working would be more helpful than that.

For some reason, my python version cannot find the `steps` folder and it stops `make test` right in it's tracks.

Reply:
Ah, I've seen this before: The steps directory isn't installed with the package, so it can't find it.
I'm not sure the "proper" solution, but you can use this command to execute the tests:
```
PYTHONPATH=. make test
```